doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -23,12 +23,61 @@ Relatedly, C-level executives and public policy professionals often make, shape,
 
 ## Enumerating Vulnerability Management Actions
-In this paper, we model the decision of “With what priority should the organization take action on a given vulnerability management work unit?” to be agnostic to whether or not a patch is available. A unit of work means either remediating the vulnerability, such as, applying a patch or deploying a mitigation. Both remediation  and mitigations often address multiple identified vulnerabilities. The deployer should answer the suggested questions for whatever unit of work they are considering as a whole, single unit. There is not necessarily a reliable function to aggregate a recommendation about remediation out of its constituent vulnerabilities. For the sake of simplicity of examples, we treat the remediation as a patch of one vulnerability, and comment on any difficulty in generalizing our advice to a more complex patch where appropriate. 
+In this paper, we model the decision of “With what priority should the organization take action on a given vulnerability management work unit?” to be agnostic to whether or not a patch is available. 
+A unit of work means either remediating the vulnerability, such as applying a patch or deploying a mitigation. Both remediation and mitigations often address multiple identified vulnerabilities. 
 
-To further clairify terms, "Remediaton occurs when the vulnerability is eliminated or removed. Mitigation occurs when the impact of the vulnerability is decreased without reducing or eliminating the vulnerability." [@dodi_8531_2020, section 3.5] Examples of remediation includes, applying patches, fixes and upgrades; or removing the vulnerable software or system from operation. Mitigating acions may include, software configuration changes, adding firewall ACLs or otherwise limiting the system's exposure to reduce the risk of the impact of the vulnerability; or accepting the risk.
+The unit of work may be different for different stakeholders.
+The units of work can also change as the vulnerability response progresses through a stakeholder's process.
+We elucidate these ideas with the following examples.
 
-### Supplying Patches
+### Vendor Units of Work
+
+On the input side of the vendor process, vendors typically receive reports of vulnerabilities in one or more versions of their product. 
+Part of the vendor's task on initial report intake is to resolve the initial report into a set of products and versions that are affected by the reported vulnerability. 
+On the output side, vendors provide fix bundles for affected products.
+
+Our working assumption is that for SSVC purposes, the vendor's unit of work is the tuple of (vulnerability, affected product). 
```

Comment:
What about:

That is, the supplier's (vendor's) unit of work is the _vulnerability_ and the supplier's (vendor's) _product that is affected_.

_or_ instead of:
the vendor's unit of work is the tuple of
use:
the supplier's unit of work is the tuple of

So a change from "vendor" to "supplier" would flow into the addition you suggested:
That is, the supplier's unit of work is the vulnerability and the supplier's product that is affected.

Reply:
fixed in cb8c78e99e6f74839d7327d56e240382ab6da98a